bundle/Resources/translations/ezrepoforms_content_type.en.xlf

```diff
@@ -276,6 +282,12 @@
         <note>key: field_definition.ezdatetime.use_seconds</note>
         <jms:reference-file line="28">lib/FieldType/Mapper/DateTimeFormMapper.php</jms:reference-file>
       </trans-unit>
+      <trans-unit id="ed0d3cb0a19dd96c1708957f17e15204465c69b6" resname="field_definition.ezfloat.default_value">
```

Comment:
@bdunogier we don't reuse string for these content type fieldDefinition global attributes?